validator/client/validator_aggregate.go

```diff
@@ -3,6 +3,8 @@ package client
 import (
 	"context"
 	"fmt"
+	"github.com/prysmaticlabs/prysm/shared/bls"
```

Comment:
goimports

Reply:
what do you mean?

Reply:
Please run `goimports -w validator/client/validator_aggregate.go` these imports are not sorted properly.